.github/workflows/ci.yml

```diff
@@ -75,12 +131,72 @@ jobs:
     timeout-minutes: 240
     env:
       LUA_DIR: D:\a\nifi-minifi-cpp\nifi-minifi-cpp\.lua
+      WINDOWS_MINIFI_OPTIONS: >-
+        -DCMAKE_BUILD_TYPE=Release
+        -DBUILD_ROCKSDB=ON
+        -DBUILD_SHARED_LIBS=OFF
+        -DCI_BUILD=ON
+        -DCUSTOM_MALLOC=OFF
+        -DDOCKER_BUILD_ONLY=OFF
+        -DDOCKER_PUSH=OFF
+        -DDOCKER_SKIP_TESTS=ON
+        -DENABLE_ALL=OFF
+        -DENABLE_AWS=ON
+        -DENABLE_AZURE=ON
+        -DENABLE_BUSTACHE=OFF
+        -DENABLE_BZIP2=ON
+        -DENABLE_CIVET=ON
+        -DENABLE_COAP=OFF
+        -DENABLE_CONTROLLER=OFF
```

Comment:
Now that you fixed its build, I think we should enable the controller even on Windows.

Reply:
good idea, https://github.com/apache/nifi-minifi-cpp/pull/1681/commits/48ebdcec251fc04221d0cbcf750bc261eed9409f